cmake/InstallAndPackage.cmake

```diff
@@ -177,6 +177,11 @@ elseif(WIN32)
         POST_BUILD
         COMMAND "${CMAKE_SOURCE_DIR}/os/windows/sign.bat" "$<TARGET_FILE:openttd>" "${WINDOWS_CERTIFICATE_COMMON_NAME}"
       )
+    elseif(DEFINED ENV{AZURE_CODESIGN_PROFILE_NAME})
```

Comment:
I suggest we just remove the old method. We will not be using it again, and it might break without us noticing. If someone wants it, it will still be in our git history. That is a much better place to archive old code, in my opinion :)

This includes the old `sign.bat`, and everything related.

Reply:
OK, can do that.